.travis.yml

```diff
@@ -1,17 +1,25 @@
 language: ruby
+
 rvm:
   - 2.0.0
   - 2.1.0
+
 before_install:
-  - gem install bundler -v 1.6.6
+  - gem install bundler -v ${BUNDLER_VERSION}
   - cd source
+  - sed -i -e "s/\(gem 'vagrant'.*tag. \)'[^']*'\(.*\)/\1'${VAGRANT_VERSION}'\2/" Gemfile
+
 install:
-  - travis_retry bundle _1.6.6_ install --without debug
+  - travis_retry bundle _${BUNDLER_VERSION}_ install --without debug
+
 script:
-  - bundle _1.6.6_ exec rake
-gemfile:
-  - gemfiles/minimal_release.gemfile
-  - gemfiles/previous_release.gemfile
-  - gemfiles/latest_stable.gemfile
-sudo: false
+  - bundle _${BUNDLER_VERSION}_ exec rake
+
+env:
+  matrix:
+    - VAGRANT_VERSION=v1.4.3 BUNDLER_VERSION=1.6.6
+    - VAGRANT_VERSION=v1.5.4 BUNDLER_VERSION=1.5.2
```

Comment:
Before vagrant 1.5.0, bundler was not referenced in the gemspec. In 1.5.x bundler [`~> 1.5.2`](https://github.com/mitchellh/vagrant/blob/v1.5.0/vagrant.gemspec#L17) is required. This means vagrant 1.5.x cannot be built with bundler greater or equals than 1.6.0. [Here](https://travis-ci.org/ggiamarchi/vagrant-openstack-provider/jobs/49859442) is what's happen in this case.

I always do things with a reason :innocent: